web/pages/[username]/calibration.tsx

```diff
@@ -205,8 +205,27 @@ const getCalibrationPoints = (betsData: [Contract, LimitBet[]][]) => {
     if (resolution !== 'YES' && resolution !== 'NO') continue
     const resolvedYES = resolution === 'YES'
 
-    for (const bet of bets as Bet[]) {
-      if (bet.amount < 0) continue // skip sales
+    let currentPosition = 0
+
+    // bets is reversed in place to track user current position
+    for (const bet of bets.reverse() as Bet[]) {
+      const betSign = bet.outcome === 'YES' ? 1 : -1
+      const nextPosition = currentPosition + bet.amount * betSign
```

Comment:
Position calculation is wrong. It should use shares, not mana amounts (which is what bet.amount is)